paths/jobs/lock_job.yaml

```diff
@@ -0,0 +1,40 @@
+---
+summary: Lock a job
+description: If you are the job owner, you may lock a job using this API request.
+operationId: job/keys/lock
```

Comment:
I wonder whether this should be `job/lock`?

Reply:
Hm, good point

Reply:
I'll look into this tomorrow morning

Reply:
I feel like there was a reason we did it this way, but I can't remember what it was

Reply:
I looked into it again this morning and for the life of me I can't remember what our reasoning was, so I'll change it :)